.github/workflows/conformance-clustermesh.yaml

```diff
@@ -1,4 +1,4 @@
-name: ClusterMesh (ci-multicluster)
+name: Conformance Cluster Mesh (ci-clustermesh)
```

Comment:
If the trigger is changed then `.github/ariane-config.yaml` needs to be updated as well :)

Reply:
I am not sure what do you mean with update, `.github/ariane-config.yaml` selects workflows by file name. We do not change the file names here.

Reply:
In the change above, the trigger advertised in the workflow name went from `ci-multicluster` to `ci-clustermesh`, so the corresponding trigger needs to be updated in Ariane config.

Reply:
I see, I'll fix it.